python/hsml/util.py

```diff
@@ -189,3 +196,27 @@ def validate_metric_value(opt_val):
     raise TypeError(
         "Metric value is of type {}, expecting a number".format(type(opt_val))
     )
+
+
+def get_predictor_config_for_model(model):
+    if model.framework is None:
```

Comment:
I guess this is a model object, in that case you check the model is of type BaseClass etc.

Reply:
Done